Gemfile.lock

```diff
@@ -42,20 +42,13 @@ GEM
       method_source (~> 0.8.1)
       slop (~> 3.4)
     rack (1.6.0)
-    rack-protection (1.5.3)
```

Comment:
Can we remove the changes for now? At the very least they should be in a separate PR if they're not related to this functionality.